reana-ui/src/pages/badgeCreator/LauncherBadgeCreator.js

```diff
@@ -0,0 +1,297 @@
+/*
+  This file is part of REANA.
+  Copyright (C) 2023 CERN.
+
+  REANA is free software; you can redistribute it and/or modify it
+  under the terms of the MIT License; see LICENSE file for more details.
+*/
+
+import { useState } from "react";
+import {
+  Button,
+  Container,
+  Divider,
+  Form,
+  Icon,
+  Popup,
+  Segment,
+} from "semantic-ui-react";
+
+import styles from "./LauncherBadgeCreator.module.scss";
+import { CodeSnippet, Title } from "~/components";
+import { api, LAUNCH_ON_REANA_BADGE_URL } from "~/config";
+import BasePage from "~/pages/BasePage";
+import { triggerNotification } from "~/actions";
+import { useDispatch } from "react-redux";
+import { stringifyQueryParams } from "~/util";
+import { Link } from "react-router-dom";
+
+const LauncherBadgeCreator = () => {
+  const dispatch = useDispatch();
+  const [launcherData, setLauncherData] = useState({
+    url: "",
+    analysisName: "",
+    specFileName: "reana.yaml",
```

Comment:
```suggestion
    specFileName: "",
```

Given that at the beginning the form is empty?